src/recordlinker/linking/mpi_service.py

```diff
@@ -57,6 +57,21 @@ def get_block_data(
     return session.execute(expr).scalars().all()
 
 
+def fetch_person_records(
+    session: orm.Session,
+    patients: typing.Sequence[models.Patient]
```

Comment:
You're passing in the `patients` list, but not using it in the function.  Did you mean to use on that on the base statement rather than retrieving a list of every Person in the database?